File: src/net/eduard/api/server/ScoreSystem.java
package net.eduard.api.server;

import org.bukkit.entity.Player;

import net.eduard.api.lib.game.DisplayBoard;

public interface ScoreSystem {
	
	void setScore(Player player,DisplayBoard scoreboard);
	void setScoreDefault(Player player);
	DisplayBoard getScore(Player player);
	public boolean hasScore(Player player);

}

File: src/main/java/net/eduard/api/server/ScoreSystem.kt
package net.eduard.api.server

import net.eduard.api.lib.score.DisplayBoard
import org.bukkit.entity.Player


interface ScoreSystem : PluginSystem{

    fun setScore(player: Player, scoreboard: DisplayBoard)
    fun setScoreDefault(player: Player)
    fun getScore(player: Player): DisplayBoard
    fun hasScore(player: Player): Boolean

}
